lynx/src/main/java/com/github/pedrovgs/lynx/LynxView.java

```diff
@@ -181,11 +185,26 @@ public LynxConfig getLynxConfig() {
    * Uses an intent to share content and given one String with all the information related to the
    * List of traces shares this information with other applications.
    */
-  @Override public void shareTraces(String plainTraces) {
-    Intent sharingIntent = new Intent(android.content.Intent.ACTION_SEND);
-    sharingIntent.setType(SHARE_INTENT_TYPE);
-    sharingIntent.putExtra(android.content.Intent.EXTRA_TEXT, plainTraces);
-    getContext().startActivity(Intent.createChooser(sharingIntent, SHARE_INTENT_TITLE));
+  @CheckResult @Override public boolean shareTraces(String fullTraces) {
+    try {
+      shareTracesInternal(fullTraces);
+      return true;
+    } catch (RuntimeException exception1) { // Likely cause is a TransactionTooLargeException on API levels 15+.
+      try {
+        /*
+         * Limit trace size to between 100kB and 400kB, since Unicode characters can be 1-4 bytes each.
+         */
+        String truncatedTraces = fullTraces.substring(0, Math.min(100000, fullTraces.length()));
```

Comment:
Using this ``substring`` call you are going to take the first part of the traces but the most relevant one is the last part of the traces content. Am I wrong?